FebrianAditya

febrianAditya opened the pull request:
add rest API and half documentation.

apakah bisa handle error di .then() ?, karena ada beberapa yang saya handle error di .then()

febrianAditya committed:
add rest API and half documentation

febrianAditya committed:
add feature login and generate acces_token

febrianAditya committed:
add feature login and generate acces_token

dhimashary commented on server/models/todo.js:
```diff
@@ -0,0 +1,37 @@
+      // define association here
+    }
+  };
+  Todo.init({
```
title description status tolong di validasi gaboleh null dan ga boleh empty

febrianAditya committed:
add documentation

febrianAditya committed:
add validation in model Todo

FebrianAditya commented:
Collection keywords secret:
SECRET=belajargeneratetoken
Hash=10

febrianAditya committed:
add 3rd API, Authentication, Authorization. Not yet handle Error

febrianAditya committed:
not yet complete about jQuery, still bug

febrianAditya committed:
1. Add Oauth Google
2. Finishing jQuery
3. Error Handler
4. Revisi documentation
5. not yet implementation 3rdAPI in Client

FebrianAditya commented:
SECRET=belajargeneratetoken
Hash=10
CLIENT_ID=1052799407511-50jorihbhk0g4jgimq1npit96ulbg0oj.apps.googleusercontent.com
GOOGLE_SECRET=belajarOauth

dhimashary commented:
tampilan form login kurang rapi

dhimashary commented:
register client belum terintegrasi

dhimashary commented:
- ketika add todo navbarnya ilang *jadi user gabisa logout / balik liat todo list dong
- navbar diatas tidak bekerja, ketika misal gw edit form pengen balik ke list todo gabisa
- edit masih belum terintegrasi ke server *masih gagal
- delete baiknya ada konfirmasi ke user dulu beneran yakin mau delete / nggak

dhimashary commented:
google login gagal ReferenceError: payload is not defined

febrianAditya committed:
rapihin tampilan form, koneksi register client dengan server, munculin navbar pada saat edit dan add, koneksi server dan client saat edit, perbaiki google login
